mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1129,7 +1140,7 @@
     <string name="deleting_browsing_data_in_progress">Deleting browsing data…</string>
 
     <!-- Dialog message to the user asking to delete all history items inside the opened group. -->
-    <string name="delete_history_group_prompt_message">This will delete all items.</string>
+    <string name="delete_all_history_group_prompt_message">Delete all sites in \&quot;%s\&quot;</string>
```

Comment:
@mavduevskiy Can you update the comment for this string with what the parameter will be replaced with?

Reply:
Also, [you should use typographic apostrophes](https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Punctuation#Punctuation-Apostrophesandquotationmarks) instead of straight double quotes (unless there are technical limitations in Android).


```suggestion
    <string name="delete_all_history_group_prompt_message">Delete all sites in “%s”</string>
```